### ISSUE TITLE
TypeScript won't run

### ISSUE THREAD

**Issue by AArnott** [lukehaas/RunJS#79]
The home page suggests that typescript will be transpiled and run transparently. When I tried, it didn't work:

![image](https://user-images.githubusercontent.com/3548/59376030-97f43a00-8d0c-11e9-815f-6106305a5bff.png)

The Run command fails to execute my TypeScript.

The "Show transpiled code" window ironically worked fine for me at one point. It's failing now (it's just a white box).

**Comment by lukehaas** [comment]
Apologies, this may have been introduced in the latest release. 1.5.1.
I'm currently working on a big TypeScript update that will enable RunJS to display TypeScript diagnostics.

**Comment by lukehaas** [comment]
@AArnott this is now resolved in the latest release: 1.5.2. Thanks for raising it.

Regarding the transpiled code window, this will be blank when the transpiler has encountered an error.
I would appreciate your input on what you think would be an appropriate thing do display there in that scenario?

**Comment by AArnott** [comment]
Awesome. I used Help -> Check for Updates in the 1.5.1 release and on top of the 'Found updates' dialog, this one appeared:

```
[Window Title]
Error

[Main Instruction]
Error: 

[Content]
Error: ENOENT: no such file or directory, rename 'C:\Users\andarno\AppData\Local\runjs-updater\pending\temp-runjs-setup-1.5.2.exe' -> 'C:\Users\andarno\AppData\Local\runjs-updater\pending\runjs-setup-1.5.2.exe'
    at F.U [as _captureStackTrace] (C:\Users\andarno\AppData\Local\Programs\runjs\resources\app.asar\entry-bundle.js:18:295113)
    at new f (C:\Users\andarno\AppData\Local\Programs\runjs\resources\app.asar\entry-bundle.js:18:324837)
    at C:\Users\andarno\AppData\Local\Programs\runjs\resources\app.asar\entry-bundle.js:18:327926
    at e.NsisUpdater.doCheckForUpdates (C:\Users\andarno\AppData\Local\Programs\runjs\resources\app.asar\entry-bundle.js:18:15954)
    at e.NsisUpdater.checkForUpdates (C:\Users\andarno\AppData\Local\Programs\runjs\resources\app.asar\entry-bundle.js:18:12972)
    at e.NsisUpdater.checkForUpdatesAndNotify (C:\Users\andarno\AppData\Local\Programs\runjs\resources\app.asar\entry-bundle.js:18:13374)
    at e.initUpdater (C:\Users\andarno\AppData\Local\Programs\runjs\resources\app.asar\entry-bundle.js:18:265884)
    at Object.<anonymous> (C:\Users\andarno\AppData\Local\Programs\runjs\resources\app.asar\entry-bundle.js:18:244596)
    at n (C:\Users\andarno\AppData\Local\Programs\runjs\resources\app.asar\entry-bundle.js:1:203)
    at C:\Users\andarno\AppData\Local\Programs\runjs\resources\app.asar\entry-bundle.js:1:996
    at Object.<anonymous> (C:\Users\andarno\AppData\Local\Programs\runjs\resources\app.asar\entry-bundle.js:1:1007)
    at Object.<anonymous> (C:\Users\andarno\AppData\Local\Programs\runjs\resources\app.asar\entry-bundle.js:20:3)
    at Module._compile (internal/modules/cjs/loader.js:693:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:704:10)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Object.<anonymous> (C:\Users\andarno\AppData\Local\Programs\runjs\resources\electron.asar\browser\init.js:188:8)
    at Object.<anonymous> (C:\Users\andarno\AppData\Local\Programs\runjs\resources\electron.asar\browser\init.js:190:3)
    at Module._compile (internal/modules/cjs/loader.js:693:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:704:10)
    at Module.load (internal/modules/cjs/loader.js:602:32)

[OK]
```

**Comment by AArnott** [comment]
> Regarding the transpiled code window, this will be blank when the transpiler has encountered an error.
I would appreciate your input on what you think would be an appropriate thing do display there in that scenario?

If the transpiler failed, show an error in the popup window perhaps. I introduced a syntax error to test what would happen. The error showed up on the right, and the transpiled window showed code that either fixed my error, or just showed a stale result.

![image](https://user-images.githubusercontent.com/3548/59381678-40a89680-8d19-11e9-90e7-df22d00f63d9.png)

**Comment by lukehaas** [comment]
The transpiled code window will now just show an error message if your code fails to transpile.
This is available in version 1.7.0

**Comment by davidfabianok** [comment]
please add this theme
